DigitalExcellence/dex-backend
# JobScheduler unit tests

## Description

The JobScheduler layer didnt have any tests till this PR.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Open the dex-backend project
2. Run the tests within the JobScheduler.Tests project
3. Make sure they respond successfull

## Issue

Closes: #381
